docs/ingest/datalakes/formats/spatial.mdx

```diff
@@ -45,11 +45,10 @@ OPTIONS (files 's3://bucket/folder/file.geojson');
   Foreign table name. Can be any string.
 </ParamField>
 <ParamField body="files" required>
-The path to the spatial file. For instance, `s3://bucket/folder/file.geojson` if the spatial file is in Amazon S3 or
```

Comment:
All other sections have a 
```
The path of a single CSV file or [multiple CSV files](#multiple-csv-files).
```

@Weijun-H Do you know if multiple GeoJSON files is supported? If so, we should add the missing documentation here.

Reply:
This feature is not supported yet, but I have seen a related ticket at https://github.com/duckdb/duckdb_spatial/issues/191.